src/openssl_wrap.cpp

```diff
@@ -614,6 +614,61 @@ time_point _asn1TimeToTimePoint(const ASN1_TIME *time)
     return epoch + (24h * days) + (1s * seconds);
 }
 
+/**
+ * @brief Convert an ASN1_TIME instance to a time_t
+ * @param [in] time An ASN1 encoded time object
+ * @return A time_t object with the same time
+ */
+time_t _asn1TimeToTimeT(const ASN1_TIME *time)
+{
+    auto maxTimeT = std::numeric_limits<time_t>::max();
+    auto minTimeT = std::numeric_limits<time_t>::min();
+
+    int days, seconds;
+
+    // first we check if the ASN1_TIME fits into time_t. We do so by comparing it to the
+    // smallest and biggest possible time_t.
+    SSL_ASN1_TIME_Ptr minAsn1TimeT;
+    SSL_ASN1_TIME_Ptr maxAsn1TimeT;
+
+    try {
+        minAsn1TimeT = _ASN1_TIME_from_time_t(minTimeT);
+    } catch (const OpenSSLException&) {
+        // Smallest time_t is smaller than even the smallest Asn1Time.
+        // We don't need to check for lower limits in this case.
```

Comment:
When should this happen? Is this really an ignorable error? The Creation might also fail because out of memory or something similar?

Reply:
Done. I've replaced this with some other code that uses numeric comparison instead.